src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
```

Comment:
This does not get unmounted so maybe a local on close function will help reset the state?

Reply:
It is already happening, but after the closing transition :-) https://github.com/aragon/aragon/blob/6db39231a265393c6e3db418a6d13035c32fe3b3/src/apps/Permissions/AssignPermissionPanel.js#L121-L125